server/src/main/java/org/opensearch/action/admin/cluster/shards/TransportCatShardsAction.java

```diff
@@ -19,10 +19,16 @@
 import org.opensearch.common.inject.Inject;
 import org.opensearch.core.action.ActionListener;
 import org.opensearch.core.action.NotifyOnceListener;
+import org.opensearch.rest.ResponseLimitBreachedException;
+import org.opensearch.rest.ResponseLimitSettings;
```

Comment:
Can we move this to action/admin/common package to avoid dependency on rest

Reply:
Move to common.breaker